.github/workflows/build-container.yml

```diff
@@ -199,6 +199,19 @@ jobs:
       - name: Display logs
         run: make logs OPTS=''
 
+      - name: Login to quay.io (on release/tag create)
+        if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
BTW, I'd add an extra condition here to be able to test that this PR works. Few ideas:
* Specific Github label (e.g.: `force-publish-containers`)
* Keyword in PR title, PR body or comment

In that case, the container image tag should contain a specific suffix to avoid overwriting existing release tags from the `git describe` (e.g.: `v0.0.2-PR-${{ github.event.number }}`).

Reply:
Well, my suggestion was to have a way to (re)publish images without having to actually (re)push a tag or create a new release... I can think of a few situations where we might want that, but on the other hand this way we ensure that tags uniquely identify codebase snapshots.